yotta.sh

```diff
@@ -2,33 +2,25 @@
 set -e
 
 DOCKER_IMAGE=mbed/yotta
-YT_USER=yotta_user
-YT_GROUP=yotta_group
-YT_HOME=/yotta_home
 
 USER_ID=$(id -u)
 GROUP_ID=$(id -g)
-YT_WORK_DIR="$YT_HOME/work/$(basename $PWD)"
-INT_SETTINGS_DIR="$YT_HOME/.yotta"
+WORK_DIR="$PWD"
```

Comment:
What does '$PWD' resolve to? Would it not be better to be explicit and avoid stomping any existing directories?